src/next.md

```diff
@@ -7,6 +7,164 @@
 This section describes what might happen after a changeset gets into the build,
 for example a bug might be filed or a backport could be requested.
 
+* [Requesting a Backport](#requesting-a-backport)
+* [Filing a Bug](#filing-a-bug)
+* [ProblemListing or Ignoring a test](#problemlisting-or-ignore-ing-a-test)
+* [Backing Out a Change](#backing-out-a-change)
+
+## Requesting a Backport
+
+TBD
+
+## Filing a Bug
+
+When a new failure is found in the JDK a bug should be filed to describe and track the issue. Depending on your role in the OpenJDK you can either use the [Bug Report Tool](https://bugreport.java.com/) or, if you are Author in an OpenJDK Project, report the bug directly in the [JDK Bug System](https://bugs.openjdk.java.net/). Try to make the bug report as complete as possible to make it easier to triage and investigate the bug. It's not the reporter's responsibility to set a correct priority, but a good guess is always better than a default value. To help with setting the right priority consider things like how the bug impacts the product and our testing, how likely is it that the bug triggers, and how difficult is it to work around the bug if it's not fixed soon.
+
+A few things to keep in mind when filing a new bug:
+
+* Before filing a bug, verify that there isn't already a bug filed for this issue.
+* Add relevant labels like `intermittent`, `regression`, `testbug` etc.
```

Comment:
didn't we _deprecated_ `testbug` label and decided to use `noreg-self` to mark all test bugs?

Reply:
Yes we did. You're absolutely right. Will change that.